### PR TITLE
refactor: let Banner can customize

### PR DESCRIPTION
**Pull request by AlanSyue**
# Description
Since we have the same banner on all pages, I think we should let the banner component can customize each word.

## Home Page
image

## About Page
image

## Graduates Page
image

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] see stage home page banner ( https://stage.itseed.tw/ )

image

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Malik705017 — APPROVED** [review]
LGTM

File: components/molecules/BannerContainer/BannerContainer.tsx [hunk 1]
```diff
@@ -13,10 +13,20 @@ import styles from "./BannerContainer.module.scss";
 type BannerContainerProperty = {
   className?: string;
   backgroundImage?: string;
+  bigTitle?: string;
+  subTitle?: string;
```
Comment (Nick0603) [inline components/molecules/BannerContainer/BannerContainer.tsx:17]: A suggestion. I think `primaryTitle` is more intuitive.

File: components/molecules/BannerContainer/BannerContainer.tsx [hunk 2]
```diff
@@ -13,10 +13,20 @@ import styles from "./BannerContainer.module.scss";
...
+  showSummaryInfo?: boolean;
 };
 
 const BannerContainer: FC<BannerContainerProperty> = ({
   backgroundImage = "",
+  bigTitle = "",
+  subTitle = "",
+  subText = "",
+  buttonText = "",
+  showSummaryInfo = false,
```
Comment (Nick0603) [inline components/molecules/BannerContainer/BannerContainer.tsx:29]: [minor, could implement later]
Maybe we need a parameter for `onButtonClick`

**Review by Nick0603 — APPROVED** [review]
Some suggestion

File: components/molecules/BannerContainer/BannerContainer.tsx (hunk 2 above)
Comment (AlanSyue) [reply components/molecules/BannerContainer/BannerContainer.tsx:29]: Add an asana ticket into the backlog
https://app.asana.com/0/1200963218344861/1201503362836963/f

**Pull request merged**